src/main/java/com/google/sps/servlets/CalendarTokenServlet.java

```diff
@@ -25,11 +49,11 @@ protected String getRedirectUri(HttpServletRequest req) throws ServletException,
 
   @Override
   protected AuthorizationCodeFlow initializeFlow() throws IOException {
```

Comment:
Why is it initializeFlow if you don't actually initialize it here just return it?

Reply:
That method is the implementation of the abstract one from AbstractAppEngineAuthorizationCodeServlet. I can't change the name